src/components/themed/ErrorText.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import * as React from 'react'
+import { StyleSheet } from 'react-native'
+
+import { type Theme, type ThemeProps, cacheStyles, withTheme } from '../services/ThemeContext'
+import { EdgeText } from './EdgeText'
+
+type OwnProps = {
+  error?: string,
+  style?: StyleSheet.Styles
```

Comment:
This is not OK. New components should *never* accept raw CSS styles.

We actually had a meeting about this a few months ago, and we agreed to do it this way: `<EdgeText><Error>{s.strings.whatever}</Error></EdgeText>`. Basically, the `Error` component would change the text color to red, but let the `EdgeText` manage the other styles (font, size, margin, padding, etc.).

We also want a `<Warning>` for yellow text, and a `<B>` for bold text.

Reply:
Removed and fixed